src/main/java/taxi/model/Driver.java

```diff
@@ -6,6 +6,8 @@ public class Driver {
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
```

Comment:
Consider using a more secure way to store passwords, like hashing and salting. 🔒